### ISSUE TITLE
feat: Make it possible to pass sklearn metrics without `"neg_"`

### ISSUE THREAD

**Issue by auguste-probabl** [probabl-ai/skore#2607]
### Is your feature request related to a problem? Please describe.

The following works:

```python
from sklearn.datasets import make_classification
from sklearn.ensemble import RandomForestClassifier
from skore import evaluate

X, y = make_classification(random_state=0)
report = evaluate(
    RandomForestClassifier(max_depth=2, random_state=0), X, y, pos_label=1
)
report.metrics.summarize(metric='neg_mean_squared_error').frame()
```

But this doesn't:
```python
report.metrics.summarize(metric='mean_squared_error').frame()
```

### Describe the solution you'd like

Make it work

**Comment by direkkakkar319-ops** [comment]
Hi, @auguste-probabl ,I looked into the issue.
The function name `mean_squared_error` is not a scorer key . It's just the name of a Python function in `sklearn.metrics.` The scorer registry only knows the `neg_` prefixed version. 

As `sklearn` says `higher score = better score` but `mean_squared_error` is doing opposite so was named `neg_` that's why it is not found in the `scorer registry`

**Comment by direkkakkar319-ops** [comment]
For fixing it we can make a normalizer function `normalize_metric()` .With this is if the user will pass the `mean_squared_error` then we can automap it like this  `neg_version = f"neg_{metric}"` and then we can return it

If we want to do it without `neg_` that would be a robust long-term solution that is to maintain a dictionary of friendly names -scorer names

**Comment by direkkakkar319-ops** [comment]
@auguste-probabl ,should I go ahead and open a pr for this

**Comment by direkkakkar319-ops** [comment]
Hi, @auguste-probabl raised a Pr related to the issue , would appreciate you feedback .

**Comment by direkkakkar319-ops** [comment]
hi, @auguste-probabl i was looking into different issues i found out that this issue also related to one of the steps of issuw #1378 plz look into it , i would also link it to my pr if you would recommend me to do so.

**Comment by GaetandeCast** [comment]
Hey @direkkakkar319-ops, quite a lot of changes are coming to the metrics part of skore so it may not be the best thing to tackle right now. Also I'm pretty sure #2646 closes this issues.

**Comment by direkkakkar319-ops** [comment]
Thanks @GaetandeCast! Understood — I'll close #2656 then to avoid any conflict. Let me know if there's anything else I can help with once the metrics refactor settles down.

**Comment by direkkakkar319-ops** [comment]
Hi, @auguste-probabl i just want to verify that should i solve this issue with the older technique used in PR #2656 or should i find a better alternative

**Comment by auguste-probabl** [comment]
That approach works fine!

**Comment by direkkakkar319-ops** [comment]
Ok i will follow the same approach
